.github/workflows/generate-docs.yaml

```diff
@@ -79,10 +79,12 @@ jobs:
     - name: Copy Docs
       run: |
         rm -rf docs/$DOCS_DIRNAME
-        mkdir -p docs/$DOCS_DIRNAME
         mkdir -p docs/$DOCS_DIRNAME/crds
         cp -r astarte-kubernetes-operator/docs/generated/* docs/$DOCS_DIRNAME/crds
         cp -r astarte-kubernetes-operator/docs/documentation/doc/* docs/$DOCS_DIRNAME
+    - name: Enforce the usage of CalVer in the generated doc pages
+      run: |
+        sed -i 's/22\.11\.0\b/22.11.00/g' docs/$DOCS_DIRNAME/*.html
```

Comment:
Do we want to hardcode the actual value here or do we want to do a more generic regex?

Reply:
I'm not sure that defining a more generic regex would be safe as it could change other versions which are shown within the generated html. Furthermore, that hardcoded value is just another value that must be bumped when preparing a new release cycle (in addition to those that we already bump).
What do you think?